test/formats/kml/util/ImagePyramid.test.js

```diff
@@ -2,20 +2,32 @@
  * Copyright (C) 2014 United States Government as represented by the Administrator of the
  * National Aeronautics and Space Administration. All Rights Reserved.
  */
-require({
-    baseUrl: '/test/'
-}, [
-    'test/CatchTest',
+define([
     'src/formats/kml/util/ImagePyramid',
     'src/util/XmlDocument'
 ], function (
-    CatchTest,
     ImagePyramid,
     XmlDocument
 ) {
     "use strict";
-    TestCase("KmlImagePyramidTest", {
-        testValidKml: CatchTest(function () {
+    describe("KmlImagePyramidTest", function () {
+        var index = 0;
+
+        beforeEach(function() {
+            this.index = index++;
+        });
+
+        afterEach(function() {
+            if (this.index > 0)
+            {   var failed = jsApiReporter.specResults(this.index -1, 1)[0].failedExpectations;
+                console.log('failed: ', failed);
+                if (failed.length > 0)
+                {
+                    console.log('After: ', this, failed[0].message);
+                    alert('ha');
+                }
+            }
+        });
```

Comment:
This part works only with one test. When there are multiple tests it fails with exception for undefined. So if we need this mechanism, we need to update it somehow.